src/components/plots/styling/frequency/SelectCellSets.jsx

```diff
@@ -64,7 +64,7 @@ const SelectCellSets = (props) => {
         <Select
           aria-label='cell sets'
           value={{
-            key: firstLetterUppercase(config.proportionGrouping),
+            key: _.upperFirst(config.proportionGrouping),
```

Comment:
the key property here is the one that is used to display? or only to store in redux? in the case it goes to redux, why do we to this `upperFirst`?

Reply:
This is only used to display, not the value that goes to redux.